packages/storybook/src/nimble/tabs/tabs-matrix.stories.ts

```diff
@@ -16,7 +16,33 @@ import { hiddenWrapper } from '../../utilities/hidden';
 import { textCustomizationWrapper } from '../../utilities/text-customization';
 import { loremIpsum } from '../../utilities/lorem-ipsum';
 
```

Comment:
Can you make the same changes to the anchor tabs stories (both matrix and docs)?